s3-sink sns-source docs changed (#53)
linkall-labs/docs: user-manual/connectors/sink/s3.md

### Set S3 Sink Secrets 

Users should set their sensitive data **Base64 encoded** in a secret file. And **mount that secret file to 
`/vance/secret/secret.json`** when running the connector.

Replace `MY_SECRET` with your sensitive data to get the Base64-based string.

```shell
$ echo -n MY_SECRET | base64
TVlfU0VDUkVU
```

Create a local secret file with the Base64-based string you just got from your secrets.

```shell
$ vim secret.json
{
  "awsAccessKeyID": "TVlfU0VDUkVU",
  "awsSecretAccessKey": "U2VjcmV0QWNjZXNzS2V5"
}
```
| Secrets            | Description                                                          | Example                       |Required|
|:-------------------|:---------------------------------------------------------------------|:------------------------------|:-------|
| awsAccessKeyID     | `awsAccessKeyID` is the Access key ID of your aws credential.        | "BASE64VALUEOFYOURACCESSKEY=" |**YES** |
| awsSecretAccessKey | `awsSecretAccessKey` is the Secret access key of youraws credential. | "BASE64VALUEOFYOURSECRETKEY=" |**YES** |

### Run the Amazon S3 Sink with Docker

> At first, you can **pull the image** of S3 Sink from dockerhub like following command.

Reviewer: ```suggestion
You can get the s3 Sink image from docker with the following command.
```

```shell
docker pull vancehub/sink-aws-s3:latest
```

> Create your `config.json` and `secret.json`, and mount them to specific paths to run the S3 Sink using following command.

Reviewer: ```suggestion
After creating your 'config.json' and 'secret.json'  files, you can run the S3 Sink and mount the files with the following command.
```

```shell
docker run -v $(pwd)/secret.json:/vance/secret/secret.json -v $(pwd)/config.json:/vance/config/config.json -p 8080:8080 --rm vancehub/sink-aws-s3
```

### Verify the Amazon S3 Sink

> To verify the S3 Sink, you should send CloudEvents to address of S3 Sink. Try to use following `curl` command. 

Reviewer: ```suggestion
To verify the S3 Sink, you should send CloudEvents to the address of S3 Sink. Try to use the  following `curl` command. 
```

```shell 
[root@iZ8vbhcwtixrzhsn023sghZ ~]# curl -X POST \
> -d '{"specversion":"0.3","id":"b25e2717-a470-45a0-8231-985a99aa9416","type":"com.github.pull.create","source":"https://github.com/cloudevents/spec/pull/123","time":"2019-07-04T17:31:00.000Z","datacontenttype":"application/json","data":{"much":"wow"}}' \
> -H'Content-Type:application/cloudevents+json' \
> http://${host ip}:8080
``` 

To fill the `${host ip}`, you can use your host IP and the port which S3 Sink docker image exposes, or you can use the 
`localhost` to fill it.

After S3 Sink Connector, you can see following logs:

```shell 
[root@iZ8vbhcwtixrzhsn023sghZ ~]# curl -X POST \
> -d '{"specversion":"0.3","id":"b25e2717-a470-45a0-8231-985a99aa9416","type":"com.github.pull.create","source":"https://github.com/cloudevents/spec/pull/123","time":"2019-07-04T17:31:00.000Z","datacontenttype":"application/json","data":{"much":"wow"}}' \
> -H'Content-Type:application/cloudevents+json' \
> http://8.142.113.12:8080

> receive CloudEvent success

Author: The message is a response replied by S3 Sink and it doesn't have 's'.
